embedded-redis/src/main/java/com/playtika/test/redis/EmbeddedRedisBootstrapConfiguration.java

```diff
@@ -69,7 +70,8 @@ public GenericContainer redis(ConfigurableEnvironment environment,
                         .withExposedPorts(properties.port)
                         .withEnv("REDIS_USER", properties.getUser())
                         .withEnv("REDIS_PASSWORD", properties.getPassword())
-                        .withCommand("redis-server", "--requirepass", properties.getPassword())
+                        .withCopyFileToContainer(MountableFile.forClasspathResource(properties.confFile), "/data/redis.conf")
```

Comment:
we need to have test on this, otherwise how we can be sure that conf file is being applied?